test/mod/test_fmt_v2.py

```diff
@@ -19,6 +19,9 @@
                 "version": "NaN",
             },
         ],
+        "exports": [
+            "build"
```

Comment:
Maybe a more realistic example here? Typically this would be "image" or "qcow2", right?